packages/flutter/lib/src/services/system_navigator.dart

```diff
@@ -10,16 +10,25 @@ import 'system_channels.dart';
 class SystemNavigator {
   SystemNavigator._();
 
-  /// Instructs the system navigator to remove this activity from the stack and
-  /// return to the previous activity.
+  /// Removes the topmost Flutter instance, presenting what was before
+  /// it.
   ///
-  /// On iOS, calls to this method are ignored because Apple's human interface
-  /// guidelines state that applications should not exit themselves.
+  /// On Android, removes this activity from the stack and returns to
+  /// the previous activity.
   ///
-  /// This method should be preferred over calling `dart:io`'s [exit] method, as
-  /// the latter may cause the underlying platform to act as if the application
-  /// had crashed.
-  static Future<void> pop() async {
-    await SystemChannels.platform.invokeMethod<void>('SystemNavigator.pop');
+  /// On iOS, calls `popViewControllerAnimated:` if the root view
+  /// controller is a `UINavigationController`, or
+  /// `dismissViewControllerAnimated:completion:` if the top view
+  /// controller is a `FlutterViewController`.
+  ///
+  /// The optional [animated] parameter is ignored on all platforms
```

Comment:
nit: Enclose animated in ` instead of [.

Reply:
Done